app/services/super_admin/access.rb

```diff
@@ -0,0 +1,19 @@
+module SuperAdmin
+  class Access
+    attr_reader :user
+
+    def initialize(user)
+      @user = user
+    end
+
+    def can_access?
+      "super_admin".in?(dfe_roles)
+    end
+
+  private
+
+    def dfe_roles
+      user&.user&.dfe_roles&.map(&:role_type)
+    end
```

Comment:
Might make sense to add a `super_user?` method to [Sessions::Users::DfEUser](https://github.com/DFE-Digital/register-early-career-teachers-public/blob/main/app/services/sessions/users/dfe_user.rb) and the correpsonding [Persona](https://github.com/DFE-Digital/register-early-career-teachers-public/blob/main/app/services/sessions/users/dfe_persona.rb).